README.rst

```diff
@@ -44,7 +44,7 @@ located in the ``s3test_boto3`` directory.
 
 You can run only the boto3 tests with::
 
-	S3TEST_CONF=your.conf tox -- -m 'not fails_on_rgw' s3tests_boto3/functional
+	S3TEST_CONF=your.conf tox s3tests_boto3/functional
```

Comment:
are you sure it doesn't still need the `--`? the path is for the `pytest` command line, not `tox`

Reply:
at least in my environment, it works either way. I think it's better without the `--` because it's consistent with the section about running a directory of tests:
https://github.com/ceph/s3-tests/blob/master/README.rst?plain=1#L23-L26

Reply:
```
~/s3-tests $ S3TEST_CONF=s3tests.conf.SAMPLE tox s3tests_boto3/functional
usage: tox [-h] [--colored {yes,no}] [-v | -q] [--exit-and-dump-after seconds] [-c file] [--workdir dir] [--root dir] [--runner {virtualenv}] [--version] [--no-provision [REQ_JSON]] [--no-recreate-provision] [-r] [-x OVERRIDE]
           {run,r,run-parallel,p,depends,de,list,l,devenv,d,config,c,quickstart,q,exec,e,legacy,le} ...
tox: error: unrecognized arguments: s3tests_boto3/functional
hint: if you tried to pass arguments to a command use -- to separate them from tox ones
```

Reply:
changed it, maybe we should change https://github.com/ceph/s3-tests/blob/master/README.rst?plain=1#L23-L26 as well since it's the same command? what do you think?

Reply:
yes please!

Reply:
done